bitcoin-review-podcast/lightning-privacy-splice-panel.md

```diff
@@ -2047,39 +1922,39 @@ But we can take more dangerous approaches to play with cryptography for the goss
 And it's okay if we have to change it entirely afterwards because it's broken.
 So it's a good ground for experimentation, but we don't want to waste too much time either on experimenting with too many things that we know won't work in the long run.
 
-Speaker 0: 01:40:16
+NVK: 01:40:16
 
 There you go.
 So see if you are a researcher who can make a production grade cryptographic library apply for a grant, you know, that's what OpenSats is for.
 
-Speaker 3: 01:40:27
+Vivek: 01:40:27
 
-Just please focus on MuSig too.
+Just please focus on MuSig, too.
 Forget about all this ZKP stuff.
 
-Speaker 0: 01:40:31
+NVK: 01:40:31
 
-I am done in MuSig too.
+I am done in MuSig, too.
 I am a Frost evangelist now.
 A fully Frost sold Frost maximalist.
 
-Speaker 4: 01:40:38
+Dusty Damon: 01:40:38
 
 Are you Roast sold though?
 
-Speaker 0: 01:40:39
+NVK: 01:40:39
 
 I mean, the Roast can get interested.
```

Comment:
We want to incorporate a glossary into the Editor that will help you better understand technical terms/concepts that you might not know.
In this case this is [ROAST](https://eprint.iacr.org/2022/550.pdf) and [FROST](https://btctranscripts.com/tabconf/2022/2022-10-14-roast/).